cloudigrade/util/aws.py

```diff
@@ -543,3 +564,30 @@ def extract_sqs_message(message, service='s3'):
     for record in message_body.get('Records', []):
         extracted_records.append(record[service])
     return extracted_records
+
+
+def rewrap_aws_errors(original_function):
```

Comment:
This file is starting to get a little long, maybe its time to turn aws.py into an aws module? We clearly have different components going into this so it wouldn't be hard to split it up by which component we're interacting with.

Reply:
We'll call it `boto4`

Reply:
The file may be getting a little big, but half of it is docstrings (we're so good with docstrings!) and I don't yet see a clear enough delineation of functionality where we would split it. We can certainly keep an eye on in for the future, though.

Reply:
My main concern really is organization, its becoming harder to scroll through and see what functions are available especially if not routinely working in this file. Maybe not now, but soon™ I'd love to have it split out into `aws.sts`, `aws.s3`, `aws.ec2`, `aws.sqs`, etc. so we have a clear understanding of what part of aws each function deals with.